packages/app/src/App.tsx

```diff
@@ -55,6 +55,7 @@ function App(props: Props) {
     >
       <ReflexContainer
         className={styles.root}
+        data-fullscreen-root
```

Comment:
Easier than a ref, since I can query the element when the user clicks on the fullscreen button.